packages/shared-core/src/helpers/rowsHelper.ts

```diff
@@ -0,0 +1,67 @@
+import { Row } from "@budibase/types"
+
+export function cleanExportRows(
```

Comment:
This function name isn't very clear, it appears this returns a bunch of rows in a format - its not just cleaning.